test/features/deploy_kogito_build.feature

```diff
@@ -47,28 +48,29 @@ Feature: Deploy Kogito Build
 
     @springboot
     Examples:
-      | runtime    | example-service            | native   | profile |
-      | springboot | process-springboot-example | disabled | default |
```

Comment:
Just FYI there actually is a `default` profile in the parent POM of kogito-examples:
https://github.com/kiegroup/kogito-examples/blob/stable/pom.xml#L119
But it seems to be used just for building purposes to limit which modules are being built. So when we are running just a concrete example, we should be fine. Just something to have in mind for the future.